### PR TITLE
Fix typo in nvcc_wrapper

### PR DESCRIPTION
**Pull request by Belcourt**

### CODE REVIEW

**Comment by mhoemmen** [comment]
Kokkos does not permit merges into master; all PRs must be based against develop.  I changed the base branch accordingly.

File: bin/nvcc_wrapper [hunk 1]
```diff
@@ -171,7 +171,7 @@ do
     shift
     ;;
   #Handle known nvcc args
-  --dryrun|--verbose|--keep|--keep-dir*|-G|--relocatable-device-code*|-lineinfo|-expt-extended-lambda|--resource-usage|-Xptxas*)
+  --dryrun|--verbose|--keep|--keep-dir*|-G|--relocatable-device-code*|-lineinfo|--expt-extended-lambda|--resource-usage|-Xptxas*)
```
Comment (mhoemmen) [inline bin/nvcc_wrapper]: If I remember right, this is not a typo.  I think `nvcc` takes this argument with either one or two hyphens.

**Comment by mhoemmen** [comment]
@Belcourt Wait, hold on, did this change actually fix something for you?

File: bin/nvcc_wrapper (hunk 1 above)
Comment (dhollman) [reply bin/nvcc_wrapper]: I think the typo is that *we* don't accept either one or two hyphens, right? so it should be something like:

```
--dryrun|--verbose|--keep|--keep-dir*|-G|--relocatable-device-code*|-lineinfo|*-expt-extended-lambda|--resource-usage|-Xptxas*)
```

**Comment by crtrott** [comment]
Couple things: 
1) we need the -expt-extended-lambda (one minus) too. NVCC takes both options so this was no typo as Mark said.
2) the pull request should go against the develop branch in kokkos/nvcc_wrapper .

**Comment by dhollman** [comment]
1. @crtrott my comment addresses this. It is a typo in that we accept fewer flags than nvcc in this respect.

**Comment by dalg24** [comment]
> 2. the pull request should go against the develop branch in kokkos/nvcc_wrapper .

@crtrott Is that documented anywhere?  Perhaps we could add a README in `bin/`.

**Pull request closed without merging**